src/app/index.tsx

```diff
@@ -30,13 +32,17 @@ const App: React.FunctionComponent = () => (
     <PollingContextProvider>
       <LocalStorageContextProvider>
         <NetworkContextProvider>
-          <Router getUserConfirmation={noop}>
-            <AppLayoutContextProvider>
-              <AppLayout>
-                <AppRoutes />
-              </AppLayout>
-            </AppLayoutContextProvider>
-          </Router>
+          <AppContextProvider>
```

Comment:
One day I'll write a `<ProviderProvider />`, I feel like every time I say that I'm joking a little less.